Drop-In/build.gradle

```diff
@@ -65,14 +65,14 @@ dependencies {
     implementation 'androidx.annotation:annotation:1.1.0'
     implementation 'androidx.fragment:fragment:1.3.5'
 
-    api 'com.braintreepayments.api:braintree-core:4.1.0'
-    api 'com.braintreepayments.api:three-d-secure:4.1.0'
-    api 'com.braintreepayments.api:paypal:4.1.0'
-    api 'com.braintreepayments.api:venmo:4.1.0'
-    api 'com.braintreepayments.api:google-pay:4.1.0'
-    api 'com.braintreepayments.api:card:4.1.0'
-    api 'com.braintreepayments.api:data-collector:4.1.0'
-    api 'com.braintreepayments.api:union-pay:4.1.0'
+    api 'com.braintreepayments.api:braintree-core:4.4.0'
```

Comment:
Let's call out that we use core v4.2.0 modules in the CHANGELOG

Reply:
Done!